src/java/com/sarality/app/view/list/ListRowViewCache.java

```diff
@@ -29,7 +29,7 @@
    * @return The element with the given Id.
    */
   public View getViewById(View rowView, int viewId) {
```

Comment:
Let's not do this please. We want people to explicitly add things to view rather than things getting added by accident.